core/templates/pages/exploration-player-page/learner-experience/tutor-card.component.ts

```diff
@@ -278,6 +320,17 @@ export class TutorCardComponent {
   getInputResponsePairId(index: number): string {
     return 'input-response-pair-' + index;
   }
+
+  @HostListener('document:click', ['$event'])
```

Comment:
Few questions regarding the use of HostListener:

1. When is this listener attached to the DOM and when is it removed?
2. Are there any performance concerns in using HostListener? It seems like we only want to detect click events when the animation is shown. Seems excessive to add an event listener on the document because it can lead to unintended side effects. (Might be an interesting thread https://github.com/angular/angular/issues/7626.)

/cc @srijanreddy98 @ashutoshc8101 in case they have any thoughts

Reply:
Used Renderer2 to add the event listener when the animation begins, and remove it as soon as the animation ends.